src/new-client/src/types/Side.ts

```diff
@@ -0,0 +1,4 @@
+export enum Side {
```

Comment:
I removed this folder because it's a duplicate of the Direction enum we have in the trades service that we can use instead.